### ISSUE TITLE
Avatar ignores with and height in contact_basic.xml

### ISSUE THREAD

**Issue by sanmiguel2019** [tinode/tindroid#95]
`contact_basic.xml`

Avatar always occupies all the height of action bar. From a design standpoint, it looks very ugly, and I don't think that's what it is meant to be because `contact_basic.xml `contains

```
android:layout_width
android:layout_height
```

but these values are being ignored in ImageView. I spent already 2 days trying to decrease avatar's size. Tried to add margins, paddings to ImageView, ImageSwitcher; tried to change layout_width, layout_height. I see changes in Android Studio:

![carol1](https://user-images.githubusercontent.com/54183931/93593744-8c320180-f9ac-11ea-83a6-850a5656d0c2.png)

 but in the app avatar always occupies all the height. Posted a question on stackowerflow but given solution works either.

I don't know if this problem will be fixed. If not, could tell me someone how can I change avatar's size for my design? Thanks.

![carol](https://user-images.githubusercontent.com/54183931/93593526-32313c00-f9ac-11ea-94e3-f3bbbe58bb42.png)

**Comment by or-else** [comment]
Here is what I see on my pixel phone:

![Screenshot_20200918-082011](https://user-images.githubusercontent.com/5938143/93615289-0f8d2c00-f988-11ea-8c38-cabd6346b4f4.png)

In general, this is not a place to get help with general programming questions. Please use Stackoverflow or pay someone to do it for you.

**Comment by sanmiguel2019** [comment]
On my 2 Android phones avatar occupies all the height of the action bar.

**Comment by or-else** [comment]
Then please debug it and post your solution here. Thanks.

**Comment by sanmiguel2019** [comment]
I give up. I have spent a lot of time trying to decrease dimensions of the avatar in the topic. No success. I can do anything but it always occupies all the height of the action bar. Tried to add margins, paddings, tried to change layout_width/height. Tried to combine RelativeLayout with LinearLayout. Tried to add `android:scaleType="fitXY"`, `android:scaleType="centerInside",` `android:adjustViewBounds="true"` Any changes have absolutely no effect - avatar always stays in the same place of the same size.

I also downloaded Tindroid from Play Store but I see the same problem on both - Android 8.1.0 and Android 5.1.1. phones.

I hope that I am trying to make changes to the correct file - its contact_basic.xml? I can not find any other corresponding files.
I also tried co change avatar dimensions in contact.xml file and it works without any problems, but in contact_basic.xml not.
